Update Footer & Update Contentcontainer

PascalGeit opened the pull request:
- Footer begrenzung (schon wieder)
- begrenzun auf allen seiten (schon wieder)
- Footer Content
- Footer Farben (schon wieder)

Silenz-coder commented on components/page/FooterComponent.tsx:
```diff
@@ -14,87 +16,53 @@ export default function FooterComponent() {
                                 </li>
                                 <li>
-                                    <a href="https://discord.gg/4eeurUVvTy" className="hover:underline">Discord</a>
+                                    <a href="https://discord.gg/4eeurUVvTy" className="hover:underline">Ausloggen</a>
```
Links anpassen

Silenz-coder commented on components/page/FooterComponent.tsx:
```diff
@@ -14,87 +16,53 @@ export default function FooterComponent() {
                     <div className="grid grid-cols-2 gap-8 sm:gap-6 sm:grid-cols-3">
                         <div>
-                            <h2 className="mb-6 text-sm font-semibold text-gray-900 uppercase dark:text-white">Resources</h2>
+                            <h2 className="mb-6 text-sm font-semibold  uppercase text-gray-700">Links</h2>
```
Die CSS-Klasse {mb-6 text-sm font-semibold  uppercase text-gray-700} wird mehrfach verwendet, bitte auslagern

PascalGeit committed:
Update Footer
Update Contentcontainer

- Footer begrenzung (schon wieder)
- begrenzun auf allen seiten (schon wieder)
- Footer Content
- Footer Farben (schon wieder)
